app/routes/task_routes.py

```diff
@@ -1 +1,132 @@
-from flask import Blueprint
\ No newline at end of file
+import os
+from flask import Blueprint, abort, make_response, request, Response
+from app.models.task import Task
+from datetime import datetime, timezone
+from ..db import db
+import requests
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+def validate_task(task_id):
+    
+    try:
+        tid = int(task_id)
+    except ValueError:
+        abort(make_response(
+            {"message": f"Task id '{task_id}' is invalid. Must be an integer."},
+            400
+        ))
+
+    task = db.session.get(Task, tid)
+    if not task:
+        not_found = {"message": f"Task with id '{task_id}' not found."}
+        abort(make_response(not_found, 404))
+
+    return task
+
+@tasks_bp.post("")
+def create_task():
+    request_body = request.get_json() or {}
+    if "title" not in request_body or "description" not in request_body:
+        return make_response({"details": "Invalid data"}, 400)
+
+    new_task = Task.from_dict(request_body)
+    db.session.add(new_task)
+    db.session.commit()
```

Comment:
Like I previously mentioned, I'd prefer you implement a helper method like [create_model](https://github.com/Ada-C23/flasky/blob/c23-live-code/app/routes/route_utilities.py#L20-L30) to reduce repetition and make this route more concise. 

```suggestion
    new_task = create_model(Task, request_body)
```